### PR TITLE
Add Gujarati, Hausa, Kannada and Odia and updated Marathi

### PR DESCRIPTION
**Pull request by kulkarnip150**

### CODE REVIEW

**Review by Richienb — CHANGES_REQUESTED** [review]
All of the files should be uppercase

**Comment by nithinkodadhavadee** [comment]
@Richienb What exactly do you mean by "All files should be in uppercase"? 
And what should the file contain, few of the files @kulkarnip150 added has the translation of "Hello World" in the language and few have "Hello World" typed in the language script(seems like what a Google Translate would do).

File: README.md [hunk 1]
```diff
@@ -21,11 +21,15 @@ Make sure to see [CONTRIBUTING.md](/CONTRIBUTING.md) for instructions on contrib
 * [4Thdimension](%23/4thdimension.4dd)
 * [8Th](%23/8th)
 * [~English](%23/~English.ext)
+* [~Gujarati](%23/~Gujarati.ext)
 * [~Hindi](%23/~Hindi.ext)
 * [~Indonesia](%23/~Indonesia.ext)
 * [~Malay](%23/~Malay.ext)
 * [~Marathi](%23/~Marathi.ext)
 * [~Sanskrit](%23/~Sanskrit.ext)
+* [~Hausa](%23/~hausa.ext)
```
Comment (Richienb) [inline README.md:30]: As you can see, the linked files are lowercase.

**Comment by Richienb** [comment]
@kulkarnip150 My bad. I mean the first letter needs to be capitalised.

**Review by kulkarnip150 — COMMENTED** [review]
Capitalize all file names and in readme.md file also

**Pull request merged**